scenes/game/game.gd

```diff
@@ -821,6 +832,9 @@ func _update_clocks():
 	if observer_mode: return
 	$PlayerLife.set_clock(player_clock_remaining)
 	$OpponentLife.set_clock(opponent_clock_remaining)
+	if enforce_timer and player_clock_remaining <= 0:
+		game_wrapper.trigger_timeout(Enums.PlayerId.PlayerId_Player, 
```

Comment:
As I mentioned before about the overtime per move time, this is where you'll have to update the logic here.
My thought is wherever in the code it switches whose time is counting, it resets the overtime timer back to the full overtime amount.
When overtime amount hits 0, then you lose for real.

You'll also need to update the display.
I think after the full time runs out, it should look like 0 (00:18) or something with 0 being the main clock that is now at 0 and the 19 being the overtime time remaining.
Stretch goals: Some reasonable sound effects for running out of main clock time and then when you get down to the last 5 seconds of the overtime time (with associated flashing?)
You don't have to add these things but I think they would be cool.

Reply:
overtime is implemented, let's call the rest to-do items